### PR TITLE
Added "How to Think About Speed Tools" article.

### PR DESCRIPTION
**Pull request by malchata**
This PR contains new content mocked up for /web in the performance section entitled "How to Think About Speed Tools" requested by @addyosmani.

### CODE REVIEW

**Comment by malchata** [comment]
I signed it!

**Comment by malchata** [comment]
Word. I'll keep watch here for further guidance/content. I think if we do an intro paragraph, I'll move the download button to below that and above the first heading.

**Comment by addyosmani** [comment]
@petele Hey Pete. We're hoping to include a button for downloading a PDF of our speed-tools infographic as part of this article. Is there a clean way for us to track downloads/clicks using the existing GA setup?

**Comment by petele** [comment]
@addyosmani - you can track analytic events for hyperlinks (assuming you're going to use that for the PDF download) via the following:
```
<a href="/internal/publishing/time-travel"
   class="gc-analytics-event"
   data-category="webFu"
   data-label="unique-value-here">
    Now you can integrate Event Tracking!
</a>
```

Please use the `webFu` category, and for `data-label`, you can use anything you want that's unique.

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2018-02-26 #}
+{# wf_published_on: 2018-02-26 #}
+{# wf_blink_components: N/A #}
+
+<style>
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: is all of this styling necessary? is it possible to use the existing DevSite styles?

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+      max-width: 97.5%;
+    }
+  }
+</style>
+
+# How To Think About Speed Tools {: .page-title }
+
+## Common myths about performance
+
+<div class="text-and-image">
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Is it possible to stick with primarily markdown? Using HTML & Markdown makes future maintenance much more of a challenge.

Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Please use the default DevSite styles and syntax here. For example:
```
<style>
  img.myth {
    width: 128px;
  }
  h3.myth {
    margin: 0;
  }
  p.myth {
    margin: 0;
  }
</style>

![A piece of paper with a line graph on it.](images/line-graph.svg){: .attempt-left .myth }

### Myth 1 {: .myth }

**User experience can be captured with a single metric.** <br>
Good user experience is not captured by a single point in time. It's composed
of a series of key milestones in your users' journey. Understand the different
metrics and track the ones that are important to your users' experience.
{: .myth }

<div class="clearfix"></div>
```

This'll make future maintenance much easier and ensures a consistent look across all of our docs.

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+
+## Understanding lab vs. field data
+
+### Lab data
+
+<img src="images/tech.svg" class="attempt-right" alt="A graphical depiction of several people getting around and interacting in a busy public setting while using mobile devices.">
+
+When performance data is collected within a controlled environment with predefined device and network settings. This offers reproducible results and debugging capabilities to help identify, isolate, and fix performance issues.
+
+#### Strengths
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Optional - change to `#### Strengths {: .compare-better }`

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+
+- Helpful for debugging performance issues
+- End-to-end and deep visibility into the user experience
+- Reproducible testing and debugging environment
+
+#### Limitations
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Optional - change to `#### Limitations {: .compare-worse }`

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+
+- Restricted set of metrics
+- Limited debugging capabilities
+
+Note: Tools like Speed Scorecard, [Chrome UX Report](/web/tools/chrome-user-experience-report/), and the Page Speed Insights speed score report this.
+
+## What are the different performance tools?
+
+<div class="tools">
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Please use standard DevSite styles here. For example - change to:
```
### Lighthouse {: .myth }

![lighthouse](images/tool-lighthouse.svg){: .attempt-left .myth }

Gives you personalized advice on how to improve your website across
performance, accessibility, PWA, SEO, and other best practices.

<div class="clearfix"></div>
```

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+      <img src="images/tool-devtools.svg" alt="Chrome Developer Tools">
+    </div>
+    <h3>Chrome Developer Tools</h3>
+    <p>Allows you to profile the runtime of a page, identify, and debug performance bottlenecks.</p>
+  </div>
+</div>
+
+## So you're a...
+
+<div class="text-and-image">
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Please use standard DevSite styles here. For example:
```
#### Marketer or developer trying to build a business case for improving user experience of your website.

![Icon of a book showing charts and graphs.](images/icon-business.svg){: .attempt-right .myth }

You speak dollars and cents and are looking for monetary figures that can help
you quantify the opportunity cost and expected lift.

* Use the **Speed Scorecard** to see how your mobile site speed compares
  against your  peers in more than 10 countries. Scores are based on real
  world data from Chrome User Experience Report.
* Use the **Impact Calculator** to estimate the potential revenue opportunity
  of improving your mobile site speed. Impact is driven by benchmark data
  from Google Analytics.
* Use **TestMySite** to test your page’s mobile loading time alongside
  industry benchmarks and to learn how simple fixes can speed up your site
  and decrease visitor loss.
```

File: src/content/en/fundamentals/performance/speed-tools/index.md (hunk 1 above)
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Please override or add as **few** additional styles as possible.

File: src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...rcle55" class="st13" cx="234.6" cy="134.1" r="2.5"/><circle id="circle57" class="st12" cx="39.1" cy="143.3" r="1.4"/><circle id="circle59" class="st12" cx="52.2" cy="114.7" r="1.4"/><circle id="circle61" class="st12" cx="61" cy="107.3" r="1.4"/><circle id="circle63" class="st12" cx="98.1" cy="74.2" r="1.4"/><circle id="circle65" class="st12" cx="215.4" cy="78.2" r="1.4"/><circle id="circle67" class="st12" cx="271.1" cy="137.4" r="1.4"/><circle id="circle69" class="st12" cx="74.6" cy="69.9" r="1.4"/><circle id="circle73" class="st12" cx="199.8" cy="138" r="1.4"/><circle id="circle75" class="st12" cx="242.7" cy="102.5" r="1.4"/></g><g id="layer3"><path id="path11" class="st10" d="M133.5 32.5S67 23.2 66.8 23.2c-26 17-38 35-44 44L133.4 51l.1-18.5z"/></g><g id="layer6"><path id="polygon37" class="st14" d="M168.2 64.2H151v88.1l20.7 9.2z"/></g><g id="layer5"><path id="path79" class="st15" d="M108.7 176.6c21 0 21-13.6 42-13.6s21 13.6 42 13.6c17.8 0 20.5-9.8 33.8-12.8-18.1-9.9-45.1-16.2-75.3-16.2S93.6 154 75.5 164c12.8 3.1 15.6 12.6 33.2 12.6z"/><path id="path81" class="st16" d="M108.8 176.4c21 0 21-13.6 42-13.6h.4l-.2-15c-30 0-57 6.2-75 16 12.5 3.2 15.4 12.6 32.8 12.6z"/></g></g></g></svg>
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg]: please use the existing lighthouse logo, we don't need to upload another one.

File: src/content/en/fundamentals/performance/speed-tools/images/tool-psi.svg [hunk 8]
```diff
@@ -0,0 +1 @@
...12.57 0 0 0 12.54-12.54v-1a12.57 12.57 0 0 1-12.54 12.54z"/><path id="_Path_11" data-name="&lt;Path&gt;" class="cls-9" d="M186.54 145.17a2.06 2.06 0 0 0 .4-1.22 2.1 2.1 0 0 0-2.09-2.09 2.05 2.05 0 0 0-1.29.46L138 177.39a12.53 12.53 0 1 0 18.37 16.41l30.16-48.65z"/><path id="_Path_12" data-name="&lt;Path&gt;" class="cls-10" d="M138 178.44l45.56-35.08a2.06 2.06 0 0 1 3.29 1.09 1.94 1.94 0 0 0 .08-.5 2.1 2.1 0 0 0-2.09-2.09 2.05 2.05 0 0 0-1.29.46L138 177.38a12.48 12.48 0 0 0-4.88 9.92v.56a12.52 12.52 0 0 1 4.88-9.42z"/><path id="_Path_13" data-name="&lt;Path&gt;" class="cls-11" d="M186.54 145.17l-30.16 48.65a12.52 12.52 0 0 1-23.22-5.93v.48a12.53 12.53 0 0 0 23.25 6.5l30.16-48.65a2.06 2.06 0 0 0 .4-1.22 1.85 1.85 0 0 0-.08-.54 2.22 2.22 0 0 1-.35.71z"/><path id="_Path_14" data-name="&lt;Path&gt;" class="cls-12" d="M186.51 142.69a2.06 2.06 0 0 1 0 2.48l-30.16 48.65a12.52 12.52 0 0 1-18.84 3l28.07 28.07h59.47a12.53 12.53 0 0 0 12.54-12.54v-18.56z"/></g><path id="_Path_15" data-name="&lt;Path&gt;" class="cls-13" d="M237.61 78.65a12.53 12.53 0 0 0-12.54-12.54H66.26a12.53 12.53 0 0 0-12.53 12.54v133.73a12.53 12.53 0 0 0 12.54 12.54h158.8a12.53 12.53 0 0 0 12.54-12.54z"/></g></g></g></svg>
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/images/tool-psi.svg]: please check if this icon exists anywhere else on the site, and use that if possible.

File: src/content/en/fundamentals/performance/speed-tools/images/tool-webpagetest.svg [hunk 9]
```diff
@@ -0,0 +1 @@
...55 8.78l-3.67 1.5a11.21 11.21 0 0 0-1.52-4.72 3.16 3.16 0 0 0-2.63-1.64q-2.37 0-3.84 3.29t-1.47 10q0 7.23 1.59 11 1.36 3.25 3.69 3.25a3.71 3.71 0 0 0 2.26-.91 8.53 8.53 0 0 0 2.09-2.49v-5.06zm10.25 14.69v-40h13.65v6.77h-9.95v8.87h9.24v6.74h-9.24v10.86h10.28v6.77zm21.95 0v-35.3h-6.06v-4.72h14.58v4.72h-6.09v35.3zm9.83 0v-40h13.3v4.72h-10.86v12.23h10.18v4.72h-10.18v13.62h11.29v4.72zm15.07-12.89l2.3-.44a19.32 19.32 0 0 0 .83 5.15 7.21 7.21 0 0 0 1.86 3 3.9 3.9 0 0 0 2.72 1.12 3.79 3.79 0 0 0 3.37-1.89 9.06 9.06 0 0 0 1.2-4.92 9.89 9.89 0 0 0-.44-3.09 5.21 5.21 0 0 0-1.32-2.17 19.34 19.34 0 0 0-3.8-2.39 11.66 11.66 0 0 1-3.7-2.77 10.09 10.09 0 0 1-1.71-3.54 17.63 17.63 0 0 1-.61-4.84 15.9 15.9 0 0 1 1.78-7.98 5.31 5.31 0 0 1 4.74-3.08 5.07 5.07 0 0 1 3.59 1.45 9.46 9.46 0 0 1 2.39 4.15 22.56 22.56 0 0 1 .91 6.36l-2.34.38a12.81 12.81 0 0 0-1.31-5.79 3.71 3.71 0 0 0-6.31-.23 7.64 7.64 0 0 0-1.09 4.27 7.4 7.4 0 0 0 .8 3.77 8.46 8.46 0 0 0 3.58 2.71 18.1 18.1 0 0 1 3.82 2.33 9.52 9.52 0 0 1 2.54 4 18.36 18.36 0 0 1 .86 6 20.48 20.48 0 0 1-.9 6.26 10 10 0 0 1-2.46 4.29 5.24 5.24 0 0 1-3.74 1.47q-3.41 0-5.44-3.66t-2.12-9.92zM235 198.55v-35.3h-6.06v-4.72h14.58v4.72h-6.09v35.3z"/></g></g></svg>
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/images/tool-webpagetest.svg]: please check if this icon exists anywhere else on the site, and use that if possible.

**Comment by addyosmani** [comment]
@malchata For the introduction to this article, let's go with the following text that we've approved as a group:

> Google has put out a lot of guidance around performance data and performance tooling. The goal of this infographic is to consolidate this guidance for developers and marketers to help them understand how to think about performance and navigate all of Google's performance tool offerings

Per the thread with Kirsten, let's move the download button a little higher up so that it's adjacent to the introduction.

File: src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg (hunk 7 above)
Comment (addyosmani) [reply src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg]: In this case, this is a version of the Lighthouse logo that includes additional treatments from our illustrator to match the outline styles found in the infographic. 

Compare new one: https://github.com/malchata/WebFundamentals/blob/4fb3722f559f7590a67596631a60089dd2d1825e/src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg

to existing logo:
https://developers.google.com/web/progressive-web-apps/images/pwa-lighthouse.png

**Comment by addyosmani** [comment]
After chatting with @petele here's where we're at with a path forward:

* It's fine to use some custom styling but he felt that we went a little overboard here and had concerns about long-term maintainability if we transition to a different setup. Let's attempt to trim down some of the custom styles used for the myths section, what are the different tools section and the non-optional sections called out. 
* For now given how soon this needs to go out soon, let's focus on what is non-optional. 
* Graphics: each icon is using a version that matches what is in the infographic. I'm going to suggest we stick with these for now.
* Let's update the download button to include the analytics setup in https://github.com/google/WebFundamentals/pull/5826#issuecomment-368908666 with a data-label of `speed-scorecard`. 

@malchata does that sound reasonable?

File: src/content/en/fundamentals/performance/speed-tools/index.md (hunk 5 above)
Comment (addyosmani) [reply src/content/en/fundamentals/performance/speed-tools/index.md]: Wrt the DevSite styles Pete's referring to, he mentioned our blue boxes are very similar to the ones at the bottom of https://developers.google.com/web/ and perhaps we should reuse the styling for those.

**Comment by malchata** [comment]
It sounds totally reasonable, @addyosmani! I think @petele's guidance is good. I'm already starting to trim what I can. There was a _lot_ of extra styling I threw in to try and match it to the PDF design, and that introduce maintainability problems for sure.

Working on these changes now!

**Comment by malchata** [comment]
Got changes in @addyosmani. LMK what you think.

**Comment by addyosmani** [comment]
Looks great so far locally. One bug I noticed was with list rendering in one of the later sections:

image

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+      padding: 0 10px;
+    }
+  }
+</style>
+
+# How To Think About Speed Tools {: .page-title }
+
+<div class="clearfix"></div>
+
+Google has put out a lot of guidance around performance data and performance tooling. The goal of this infographic is to consolidate this guidance for developers and marketers to help them understand how to think about performance and navigate all of Google's performance tool offerings
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: Please wrap lines at 80 chars!!

File: src/content/en/fundamentals/performance/speed-tools/index.md [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+
+<a class="button download" href="pdf/Infographic_HowToThinkAboutSpeedTools.pdf" class="gc-analytics-event" data-category="webFu" data-label="speed-scorecard" target="_blank">Download the PDF version</a>
```
Comment (petele) [inline src/content/en/fundamentals/performance/speed-tools/index.md]: two `class` attributes

**Comment by malchata** [comment]
- Wrapped lines to 80 chars where possible.
- Fixed double `class` attribute on download.

Not sure what to do about the floating list items/text, though. That might require markup and special styles, unless there are utility classes I'm not aware of.

**Comment by petele** [comment]
> Not sure what to do about the floating list items/text, though...

Is that one where we can put the images on the right (`.attempt-right`), even though it's a touch different from the PDF?

**Comment by addyosmani** [comment]
Would aligning them right address the list floating issues? (I would have thought not, but perhaps we can try it out). I don't think it's the end of the world if we have to attempt right.

**Comment by malchata** [comment]
`.attempt-right` seems to work well for this, even if not the most optimal of choices. Commit incoming...

**Comment by addyosmani** [comment]
I'm fine with the final version from the last commit above. We're going to circle back for a final design review just to sanity check if anything else needs to be done, but we should otherwise be good to wrap up soon.

**Comment by addyosmani** [comment]
There was some minor additional feedback from the discussion thread:

* Add a table of contents - this is already in the RHS navigation. I would double check we're reflecting all headlines correctly but it seems we're otherwise good on that front.
* Double checking link consistency between the text and PDF

**Comment by addyosmani** [comment]
We've received LGTMs on the latest version from all stakeholders via email. @petele could we land and deploy this PR on Thursday, please?

**Comment by petele** [comment]
It looks like some of the images in this PR were executable. Any idea why that might be, or if it's actually necessary? I think it's causing issues with our deployment scripts.

**Comment by malchata** [comment]
They're all SVG images. Are those not allowed in /web?

**Pull request merged**